### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Pacu2**
TODO:
- [x] waiting for a new photo instead of pirate
- [x] ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,71 @@
...
+# Get involved
 
-Commercial support
-------------------
+## ❤️  Community
 
-Disclaimer: everything you see here is open and free to use as long as you comply with the [license](LICENSE). It is not a bait to force you to pay us later and we promise to do our best to fix bugs and improve the code.
+-   Join the chat on [gitter](https://gitter.im/mirumee/saleor)
+-   Follow us on [Twitter](https://twitter.com/getsaleor?lang=en)
+-   Check our latest blog posts on [medium](https://medium.com/saleor)
```
Comment (patrys) [inline README.md]: Medium spell their name with a capital "M"

File: README.md [hunk 2]
```diff
@@ -1,60 +1,71 @@
...
 
-### Mirumee Software
+Some of them however, might not be listed yet. Check [our roadmap](https://github.com/mirumee/saleor/projects/6) or think of your own feature, then drop us a message and we will discuss the most efficient way to handle it!
 
-http://mirumee.com
+If you have any ideas, just [open an issue](https://github.com/mirumee/saleor/issues/new) and tell us what you think!
+
+Get more details in our [Contributing Guide](https://saleor.readthedocs.io/en/latest/contributing.html).
+
+##  📝  Your feedback
```
Comment (patrys) [inline README.md]: Nitpick: spacing around emojis is inconsistent

File: README.md [hunk 3]
```diff
@@ -1,60 +1,71 @@
...
 Login credentials: `admin@example.com`/`admin`
 
+# Get involved
 
-Commercial support
-------------------
+## ❤️  Community
 
-Disclaimer: everything you see here is open and free to use as long as you comply with the [license](LICENSE). It is not a bait to force you to pay us later and we promise to do our best to fix bugs and improve the code.
+-   Join the chat on [gitter](https://gitter.im/mirumee/saleor)
```
Comment (patrys) [inline README.md]: Nitpick: why three spaces after the dash? I think we typically use a single asterisk (`*`) and a space for bullet lists.

File: README.md [hunk 4]
```diff
@@ -1,60 +1,71 @@
...
+[Visit our homepage to find out more](http://getsaleor.com/)
 
-Usage
------
+# Getting started
 
-See the [Saleor docs](https://saleor.readthedocs.io) for installation and deployment instructions.
+## 💾 Installation and requirements
 
+Saleor requires Python 3.4+, Node.js, PostgreSQL and OS-specific dependency tools.
```
Comment (NyanKiyoshi) [inline README.md]: I think you should say Node.js 8.0+, especially as xenial and others are still on 4.0+ which is often error prone for many users.

**Comment by NyanKiyoshi** [comment]
> waiting for new photo instead of pirate

While you are here, you can put it in the center, using CSS, it would look way better!

**Pull request merged**